VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -352,6 +352,11 @@ def get_total_used_size(self):
                         isKnownFs = True
                         break
 
+                if int(used) < 0 :
```

Comment:
Can this change be added after line 336 instead? Any reason for running additional steps after calculating value of used variable?